src/Loop.php

```diff
@@ -275,6 +275,34 @@ public static function setErrorHandler(callable $callback = null)
     }
 
     /**
+     * Retrieve an associative array of information about the event loop driver.
+     *
+     * The returned array MUST contain the following data describing the driver's
+     * currently registered watchers:
+     *
+     *  [
+     *      "defer"         => ["enabled" => int, "disabled" => int],
+     *      "delay"         => ["enabled" => int, "disabled" => int],
+     *      "repeat"        => ["enabled" => int, "disabled" => int],
+     *      "on_readable"   => ["enabled" => int, "disabled" => int],
+     *      "on_writable"   => ["enabled" => int, "disabled" => int],
+     *      "on_signal"     => ["enabled" => int, "disabled" => int],
+     *      "keep_alive"    => int,
+     *  ];
+     *
+     * keep_alive refers to the number of referenced watchers.
```

Comment:
That's an option, but we typically don't keep separate counters for references (i.e. just an array with _all_ the referenced watchers to see whether there are still referenced ones).

But we can walk the array … shouldn't matter too much in info(). So, I'm actually fine with that.